aws/eks/eks-how-to/deploy-ubuntu-pro-cluster.rst

```diff
@@ -0,0 +1,282 @@
+Deploy an Ubuntu Pro EKS cluster
+================================
+
+This guide shows how to deploy an EKS Cluster with Ubuntu Pro nodes.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+You need:
+
+- ``eksctl``: see how to install it `here <https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html>`_
+- ``packer``: only needed if you want to enable FIPS for the cluster nodes. Install it with ``sudo snap install packer``
+- your AWS access key ID and secret access key
+- an Ubuntu Pro token
+
+
+Cluster deployment preparation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Depending on whether you need to enable FIPS or not, you only need to follow one of the following two sections.
+
+Please note that, while there are Ubuntu Pro AMIs available in AWS, at the time
+of writing this guide, there is no such offering for the EKS service, so you'll
+need to provision the EKS cluster with customized Ubuntu nodes.
+
+
+With FIPS
+^^^^^^^^^
+
+When enabling FIPS, a reboot of the underlying node is required, which if done post-cluster creation, can lead to a defective cluster.
```

Comment:
a reboot can lead to a defective cluster? If that's true, we need to fix it. Can you report a bug please?

Reply:
If we reboot a node, it can be flagged as defective sometimes. I don't think that is a bug, no?

Reply:
@toabctl is right. It might not be necessarily a bug though.

Suggestion: if you have really verified this behavior @SamirPS , let's open a separate thread where you document the entire process and provide proof of instability when rebooting nodes. I assume this instability isn't related with Ubuntu Pro, so the test environment could simply be: use cloud-init to reboot nodes, and keep scaling the cluster until it gets defective, providing proof of such.

Then we can discuss whether it is a bug or not, and who should handle that (maybe it issue is at AMI level, maybe it's infrastructure related (aka AWS) or maybe even cloud-init related...)